tests/core/test_lane.py

```diff
@@ -4627,6 +4627,87 @@ def test_works_and_works_from_database(
             from_db = fixture.work_ids_from_db(lane)
             assert from_search == from_db
 
+    def test_works_and_works_from_database_with_suppressed(
+        self,
+        work_list_groups_end_to_end_fixture: WorkListGroupsEndToEndFixture,
+    ):
+        db = work_list_groups_end_to_end_fixture.db
+        fixture = work_list_groups_end_to_end_fixture
+        index = fixture.external_search_fixture.external_search_index
+
+        # Create a bunch of lanes and works.
+        data = fixture.populate_works()
+        lane_data = fixture.create_lanes(data)
+
+        decoy_library = db.library()
+        another_library = db.library()
+
+        db.default_collection().libraries += [decoy_library, another_library]
+
+        # Add a couple suppressed works, to make sure they don't show up in the results.
+        suppressed_work1 = db.work(
```

Comment:
Minor: Suggest naming this to `globally_suppressed_work`, `licensepool_suppressed_work` or similar to make this a little easier to follow below.

Reply:
Renamed